### PR TITLE
Don't fix fscale for t distribution

### PR DESCRIPTION
**Pull request by dsaxton**
Closes https://github.com/dsaxton/resample/issues/35

As mentioned there's no reason why the variance can't be allowed to be less than one. Also renamed fit_kwds to fit_kwargs.

### CODE REVIEW

File: resample/bootstrap.py [hunk 1]
```diff
@@ -215,15 +215,15 @@ def _fit_parametric_family(dist: stats.rv_continuous, sample: np.ndarray) -> Tup
         return np.mean(sample, axis=0), np.cov(sample.T, ddof=1)
 
     if dist == stats.t:
```
Comment (HDembinski) [inline resample/bootstrap.py:217]: We can just remove the first `if`, then the `else` clause at the end handles this.

File: resample/bootstrap.py [hunk 2]
```diff
@@ -215,15 +215,15 @@ def _fit_parametric_family(dist: stats.rv_continuous, sample: np.ndarray) -> Tup
...
-        fit_kwd = {"fscale": 1}
+        fit_kwargs = {}
     elif dist in {stats.f, stats.beta}:
```
Comment (HDembinski) [inline resample/bootstrap.py:219]: The elif needs to be turned into an if then.

File: resample/bootstrap.py (hunk 1 above)
Comment (dsaxton) [reply resample/bootstrap.py]: Fixed

**Comment by HDembinski** [comment]
Looks good to me!

**Pull request merged**